src/js/views/maps/LayerCategoryListView.js

```diff
@@ -82,8 +100,21 @@ define(
          * @returns {boolean} - True if a layer item matches the text
          */
         search(text) {
-          return _.reduce(this.layerCategoryItemViews, (matched, layerCategoryItem) => {
-            return layerCategoryItem.search(text) || matched;
+          return _.reduce(this.panels, (matched, panel) => {
+            const searchResultsFound = panel.contentViewInstance.search(text);
+            if (searchResultsFound) {
+              panel.$el.show();
+              if (text !== '') {
+                panel.open();
+              } else {
+                panel.collapse();
+              }
+            } else {
+              panel.$el.hide();
+              panel.collapse();
+            }
```

Comment:
Can this behavior be moved into expansion panel? Is it generalizable?

Reply:
I don't think so, expansion panels in general shouldn't have anything to do with search. These do, because the components inside of them (LayerListView) are considered in layer search